### PR TITLE
[FIX] mail: yellow bubble when there's a mention of self

### PR DESCRIPTION
**Pull request by alexkuhn**
Before / After
before after

### CODE REVIEW

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

File: addons/mail/static/src/core_ui/message.xml [hunk 1]
```diff
@@ -94,8 +93,9 @@
                                         <div class="position-relative">
                                             <div class="o-mail-Message-bubble rounded-bottom-3 position-absolute top-0 start-0 w-100 h-100" t-att-class="{
                                                 'border': state.isEditing,
-                                                'bg-info-light border border-info opacity-25': !message.isSelfAuthored and !message.isNote,
+                                                'bg-info-light border border-info opacity-25': !message.isSelfAuthored and !message.isNote and !message.isHighlightedFromMention,
                                                 'bg-success-light border border-success opacity-25': message.isSelfAuthored and !message.isNote,
```
[the remark this answers is not in the document]
Comment (alexkuhn) [reply addons/mail/static/src/core_ui/message.xml:97]: With current conditions, self-mention on self-message will show yellow bubble.
This looks wrong, so I'm adding `!selfAuthored` in below condition.

File: addons/mail/static/src/core_ui/message.xml [hunk 2]
```diff
@@ -94,8 +93,9 @@
...
+                                                'bg-warning-light border border-warning opacity-50': message.isHighlightedFromMention and !message.isSelfAuthored,
```
Comment (alexkuhn) [inline addons/mail/static/src/core_ui/message.xml:98]: I just added `and !isSelfAuthored` here

File: addons/mail/static/src/core_ui/message.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (seb-odoo) [reply addons/mail/static/src/core_ui/message.xml:97]: For reference discord shows mention highlight even if self authored (so the opposite of what is done here)

**Comment by alexkuhn** [comment]
👆reverted so now self-mention on self-message shows yellow bubble

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**